app/api/server/v1/inquiries.js

```diff
@@ -0,0 +1,37 @@
+import { Meteor } from 'meteor/meteor';
+
+import { API } from '../api';
+
+API.v1.addRoute('inquiries.get', { authRequired: true }, {
+	get() {
+		const { department } = this.queryParams;
+
+		let inquiries;
+		Meteor.runAsUser(this.userId, () => {
```

Comment:
You need to check the user permission before getting the inquiries:
```suggestion
		if (!hasPermission(this.userId, 'view-livechat-manager')) {
			return API.v1.unauthorized();
		}
```